### PR TITLE
Add link from edit button to edit.html.erb plus page content

### PR DESCRIPTION
**Pull request by scara82**
This PR:
- Add link for edit button;
- Add view for show.html.erb to show the edited family.

### CODE REVIEW

**Comment by apalski** [comment]
LGTM!

File: app/views/families/_family.html.erb [hunk 1]
```diff
@@ -9,5 +9,5 @@
   <td><%= family.animals %></td>
   <td><%= family.active %></td>
   <td><%= link_to t("families.index.destroy"), family_path(family), method: :delete, class: "list-button" %></td>
-  <td><%= link_to t("families.index.edit"), family_path(family), class: "list-button" %></td>
+  <td><%= link_to t("families.index.edit"), edit_family_path(family), method: :get, class: "list-button" %></td>
```
Comment (elle) [inline app/views/families/_family.html.erb]: I believe `method: :get` is the default and does not need to be explicitly set.

File: app/views/families/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,15 @@
...
+  <tr>
+    <td><%= family.last_name %></td>
+    <td><%= family.first_name %></td>
+    <td><%= family.address %></td>
+    <td><%= family.capability %></td>
+    <td><%= family.preferred_gender %></td>
+    <td><%= family.prof %></td>
+    <td><%= family.bus_driver %></td>
+    <td><%= family.animals %></td>
+    <td><%= family.active %></td>
```
Comment (elle) [inline app/views/families/show.html.erb]: If this is the same as what is in the `_family` partial, a suggestion to reuse the partial...

File: app/views/families/_family.html.erb (hunk 1 above)
Comment (scara82) [reply app/views/families/_family.html.erb]: I removed it, thanks!

File: app/views/families/show.html.erb (hunk 2 above)
Comment (scara82) [reply app/views/families/show.html.erb]: Done1

**Comment by scara82** [comment]
@elle  and @apalski  thanks for the feedbacks, I'm merging now!

**Pull request merged**